Makefile

```diff
@@ -0,0 +1,5 @@
+assets:
+	cd funnel/assets && npm install && npm run build
+
+assetsonly:
+	cd funnel/assets && npm run build
```

Comment:
Add an `all:` target. This will get called when `make` is executed without any parameters. It should be used for the default action. Also, the commands can be on separate lines without using `&&` as Make will abort if any fails (the equivalent of `set -e` in a bash script).